operate/client/src/modules/mocks/api/elementInstances/fetchElementInstance.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {mockGetRequest} from '../mockRequest';
+import {GetElementInstanceResponseBody} from '@vzeta/camunda-api-zod-schemas';
+
+const mockGetElementInstance = (contextPath = '') =>
+  mockGetRequest<GetElementInstanceResponseBody>(
+    `${contextPath}/v2/element-instances/:elementInstanceKey`,
```

Comment:
@juliasaienko you can consume the endpoints from `@vzeta/camunda-api-zod-schemas` too